src/app/ssmt/explore-opportunities/explore-opportunities-form/turbine-form/explore-turbine-form/explore-turbine-form.component.html

```diff
@@ -15,50 +15,75 @@
-        <span *ngIf="baselineForm.value.useTurbine == false">Off</span>
+  <li class="explore-opps-item mb-4">
+    <form class="explore-opps-form" [formGroup]="baselineForm">
+      <div class="text-center">
+        <label for="{{turbineType+'baselineUseTurbine'}}" class="baseline-value-label"> Turbine Status</label>
+        <div class="text-muted text-center baseline-value-input ">
+          <span *ngIf="baselineForm.value.useTurbine == true">On</span>
+          <span *ngIf="baselineForm.value.useTurbine == false">Off</span>
+        </div>
       </div>
-    </div>
-  </form>
-  <form class="explore-opps-form" [formGroup]="modificationForm">
-    <div class="form-group explore-opps-formgroup">
-      <label for="{{turbineType+'modificationUseTurbine'}}" class="mod-value-label"> Turbine Status</label>
-      <select name="{{turbineType+'modificationUseTurbine'}}" id="{{turbineType+'modificationUseTurbine'}}"
-        class="form-control w-100" (focus)="focusField('useTurbine')" (change)="save()" formControlName="useTurbine">
-        <option [ngValue]="true">On</option>
-        <option [ngValue]="false">Off</option>
-      </select>
-    </div>
-  </form>
-</li>
-</div>
+    </form>
+    <form class="explore-opps-form" [formGroup]="modificationForm">
+      <div class="form-group explore-opps-formgroup">
+        <label for="{{turbineType+'modificationUseTurbine'}}" class="mod-value-label"> Turbine Status</label>
+        <select name="{{turbineType+'modificationUseTurbine'}}" id="{{turbineType+'modificationUseTurbine'}}"
+          class="form-control w-100" (focus)="focusField('useTurbine')" (change)="save()" formControlName="useTurbine">
+          <option [ngValue]="true">On</option>
+          <option [ngValue]="false">Off</option>
+        </select>
+      </div>
+    </form>
+  </li>
+  </div>
 
 <!--isentropicEfficiency-->
-<div *ngIf="modificationForm.controls.useTurbine.value == true">
+<!-- Testing with 3 header example 
+High Pressure to Low Pressure
+Isentropic Efficiency
+65
+%
+Generator Efficiency
+100
+%
+Operation Type balance header
+
+High Pressure to Medium Pressure
+Isentropic Efficiency
+65
+%
+Generator Efficiency
+100
+%
+Operation Type balance header
+-->
+<div *ngIf="modificationForm.controls.useTurbine.value == true || showIsentropicEfficiency">
   <li class="explore-opps-item">
-    <div class="checkbox mt-2" *ngIf="showBaseline">
+    <div class="checkbox mt-2">
       <label for="{{turbineType+'modifyIsentropicEfficiency'}}" class="steam-check-label">
-        <input type="checkbox" id="{{turbineType+'modifyIsentropicEfficiency'}}"
+        <!-- We want to hide the checkbox when Turbine Status is on (modificationForm.controls.useTurbine.value == true) 
+              since it's not actively changing ui during that time -->
+        <input *ngIf="modificationForm.controls.useTurbine.value == false" type="checkbox" id="{{turbineType+'modifyIsentropicEfficiency'}}"
```

Comment:
After talking with Kristina - we need to make the subcheck and header show when the user is adding modifications triggered by Turbine Status selection. This is different than the current release behavior. I'm trying to hide the check under this condition since it's inactive. Currently failing on 1 out of 4 of the shared component renders (see image).
![AdjustIsentropicEfficiency-debug1](https://user-images.githubusercontent.com/60401912/77648697-91028500-6f36-11ea-9f45-da66f7aedd75.png)

I'll need to make this change for Adjust Generator Efficiency and Modify Operation type also.